src/src/Controller/Task/UpdateTaskController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Controller\Task;
+
+use App\Entity\Task;
+use App\Form\TaskType;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bridge\Doctrine\Attribute\MapEntity;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class UpdateTaskController extends AbstractController
+{
+    #[Route('/task/update/{taskId}', name: 'app_task_update')]
+    public function index(
+        #[MapEntity(mapping: ['taskId' => 'id'])]
```

Comment:
You could probably leave out this mapping if you named the parameter `task` instead of `taskId`